src/lib/cli/pager.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+/**
+ * External dependencies
+ */
+import { spawn } from 'child_process';
+
+let proc;
+export default function pager() {
+	const args = ( process.env.PAGER || 'less -FRX' ).split( ' ' );
+	const bin = args.shift();
+
+	proc = spawn( bin, args, { stdio: [ -1, 1, 2 ] } );
```

Comment:
Minor: Can we switch up these numbers to constants?

Reply:
Yeah, good call. I had been experimenting with these while debugging. Fixed now.